crates/pop-contracts/README.md

```diff
@@ -0,0 +1,116 @@
+# pop-contracts
+
+A crate for generating, building, deploying and calling [`ink!`](https://github.com/paritytech/ink) smart contracts. 
+
+## Usage
+
+Generate a new Smart Contract:
```

Comment:
Smart contract with capitals here but not in the `pop-cli` README. We should be consistent to look professional. I suggest using the capitals like done here

Reply:
Refactored the whole README to have capitals!